cmd/kubeadm/app/apis/kubeadm/v1beta3/types.go

```diff
@@ -198,6 +180,15 @@ type ImageMeta struct {
 	//TODO: evaluate if we need also a ImageName based on user feedbacks
```

Comment:
i don't think we got any strong requests for this one.

Reply:
If we want folks to use their own `hyperkube` image, a possibility would be to remove the boolean (already discussed elsewhere), and introduce `ImageName`, so they can force `hyperkube` on the Kubernetes components, hence having a similar feature.

However, if we don't want so support this feature at all because we don't have strong evidence about users wanting this, I'm fine to drop all comments on our types (this comment comes from  other older types as well).

Reply:
imagename was mainly not supported because we don't know how to handle upgrades with it, there are also custom tags.